network.go

```diff
@@ -52,6 +52,8 @@ type NetworkController interface {
 	// Create a new network. The options parameter carry driver specific options.
 	// Labels support will be added in the near future.
 	NewNetwork(d *NetworkDriver, name string, options interface{}) (Network, error)
+	// Register an external driver.
+	RegisterDriver(name string, driver driverapi.Driver)
```

Comment:
> I dont think it is a good idea to expose the driverapi to the Northbound API towards the applications using Libnetwork.

How else is docker (as a dependent of libnetwork and the plugins subsytem) going to provide a plugin-based driver for libnetwork?